### ISSUE TITLE
Need More Documentation

### ISSUE THREAD

**Issue by ashickurnoor** [geggleto/slim-queue#2]
Can you add more documentation? I couldn't understand.

**Comment by geggleto** [comment]
Yes it is on my to-do list; this weekend I will get around to it.

**Comment by geggleto** [comment]
Hey @ashickurnoor I added an example folder and did another release.

https://github.com/geggleto/slim-queue/releases/tag/0.1

let me know if you have any questions
